specification/resources/resource-manager/Microsoft.Resources/stable/2022-05-01/changes.json

```diff
@@ -0,0 +1,420 @@
+            ]
+          }
+        },
+        "changeAttributes": {
+          "$ref": "#/definitions/ChangeAttributes"
+        },
+        "changes": {
+          "$ref": "#/definitions/ChangesDictionary"
+        }
+      }
+    },
+    "ChangeAttributes": {
+      "description": "Details about the change resource",
+      "type": "object",
+      "properties": {
+        "correlationId": {
+          "description": "The ARM correlation ID of the change resource",
+          "type": "string",
+          "readOnly": true
+        },
+        "timestamp": {
+          "description": "The time the change(s) on the target resource ocurred",
+          "type": "string",
+          "readOnly": true
+        },
+        "changesCount": {
+          "type": "integer",
+          "format": "int64",
+          "readOnly": true,
+          "description": "The number of changes this resource captures"
+        },
+        "previousResourceSnapshotId": {
+          "description": "The GUID of the previous snapshot",
+          "type": "string",
+          "readOnly": true
+        },
+        "newResourceSnapshotId": {
+          "description": "The GUID of the new snapshot",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "ChangesDictionary": {
+      "description": "A dictionary with changed property name as a key and the change details as the value",
+      "type": "object",
+      "additionalProperties": {
+        "$ref": "#/definitions/ChangeBase"
+      }
+    },
+    "ChangeBase": {
+      "description": "An individual change on the target resource",
+      "type": "object",
+      "properties": {
+        "propertyChangeType": {
+          "description": "The type of change that occurred",
+          "type": "string",
+          "readOnly": true,
+          "enum": [
+            "Insert",
+            "Update",
+            "Remove"
+          ],
+          "x-ms-enum": {
+            "name": "PropertyChangeType",
+            "modelAsString": false,
+            "values": [
+              {
+                "value": "Update",
+                "description": "An existing property underwent a change",
+                "name": "Update"
+              },
+              {
+                "value": "Insert",
+                "description": "A property was newly created",
+                "name": "Insert"
+              },
+              {
+                "value": "Remove",
+                "description": "An existing property was deleted",
+                "name": "Remove"
+              }
+            ]
+          }
+        },
+        "changeCategory": {
+          "description": "The entity that made the change",
+          "type": "string",
+          "readOnly": true,
+          "enum": [
+            "User",
+            "System",
+            "Create"
+          ],
+          "x-ms-enum": {
+            "name": "ChangeCategory",
+            "modelAsString": false,
+            "values": [
```

Comment:
>values [](http://example.com/codeflow?start=13&length=6)

Should add entry for new enum value (Create).

Reply:
This is the "typo" I was mentioning in the earlier email. That should have never made it into our swagger. In our code, we only generate either "User" or "System". This was just an error from copy + pasting the enum values.